src/components/Dashboard/Content/index.tsx

```diff
@@ -18,6 +41,38 @@ const DashboardContent = ({ ...attributes }: DashboardContentProps): JSX.Element
         <Shimmer classAttributes="w-[56px] h-[56px] rounded-full" />
       </div>
     </div>
+  ) : (
+    <div className="flex flex-col h-full" {...attributes}>
+      <div
+        style={{ backgroundImage: `url(${surveys[currentPosition].imageUrl})` }}
+        className={`w-full h-[302px] rounded-[12px] bg-cover duration-500 ease-in-out`}
+      ></div>
+      <div className="flex flex-row justify-between mt-[38px]">
+        <div className={`flex flex-col justify-between`}>
+          <p className={`text-white text-x-regular font-extrabold`}>{surveys[currentPosition].title}</p>
+          <p className="text-white text-regular tracking-survey-tight opacity-60 mt-2">{surveys[currentPosition].description}</p>
+        </div>
+        <button type="button" className="w-[56px] h-[56px] bg-white rounded-full inline-flex items-center justify-center">
+          <ArrowRight />
+        </button>
+      </div>
+      {/* <!-- Slider indicators --> */}
+      <div className="flex-1 flex space-x-3 justify-center items-end mb-[42px]">
+        {surveys.map((surveyItem, index) => {
+          return (
+            <button
```

Comment:
I think we already have a button component, right? Could we use that here instead?

Reply:
This is just a tiny button, our customized button is not designated for this so I decide to create a `<button>` tag here 🙏